src/frontend/src/utils/urls.ts

```diff
@@ -148,6 +148,9 @@ const designReviewById = (id: string) => `${designReviews()}/${id}`;
 const designReviewDelete = (id: string) => `${designReviewById(id)}/delete`;
 const designReviewMarkUserConfirmed = (id: string) => `${designReviewById(id)}/confirm-schedule`;
 
+/**************** Work Package Template Endpoints ****************/
+const workPackageTemplates = () => `${API_URL}/work-packages/template`;
```

Comment:
use the work_packages endpoint that already exists and can you move down the other work package template routes that are in the work package section down here